test/presenters/document_collection_presenter_test.rb

```diff
@@ -124,4 +124,21 @@ class GroupWithDocumentsWhenThereIsNoLinksDocuments < TestCase
       assert_empty group_with_missing_documents
     end
   end
+
+  class GroupWithDocumentsWhenThereAreWithdrawnDocuments < TestCase
+    test "does not present withdrawn documents" do
+      presenter = presented_item(
+        "document_collection_with_withdrawn_links_documents"
+      )
+
+      groups_without_withdrawn_documents = presenter.groups.first["documents"]
+
+      number_of_documents_that_have_withdrawn_false = 2
```

Comment:
Not happy with this but because both `group_documents` and `documents_hash` are private I couldn't figure out a neater way to check that the withdrawn ones have been excluded. The only downside with doing this is if somebody goes and modifies the example by say, adding another document, 99% chance is they will break these tests. However, that happens rare enough that this should be fine.